modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-preparing-container-disk-for-vms_{context}"]
+= Preparing a container disk for virtual machines
+
+A container disk must be built with a virtual machine image and pushed to a container registry. It can then either be imported into a DataVolume and attached to a virtual machine or attached directly to a virtual machine as an ephemeral containerDisk volume.
+
+.Prerequisites
+
+* You have root or sudo privileges on your local system to install podman.
+
+* The virtual machine image must be either QCOW2 or RAW format.
+
+.Procedure
+
+. Install `podman` if it is not already installed:
++
+[source,terminal]
+----
+$ sudo yum install podman -y
+----
+
```

Comment:
before that, we need to ensure that the image would be readable by qemu
```
chmod 440 <vm_image>.qcow2
```

Reply:
our test suite currently verifies with 444, not 440.

440 might work, but that isn't tested at the moment.

Reply:
I added a PR (https://github.com/kubevirt/kubevirt/pull/4416) to KubeVirt to both build and test our containerDisks using 107:107 ownership and 440 file mode instead of the previous 444 mode.

So, once we get this build/test change merged, our officially tested recommendation can be 440.

Reply:
@davidvossel Have updated the Dockerfile step and explanation based on the comments here and in #25359. Can you please confirm this is correct now?

Reply:
Followup: I got ack from David for the Dockerfile update and 25359 is now merged.